### ISSUE TITLE
2.83 not supported

### ISSUE THREAD

**Issue by smguzik** [DarklightGames/io_scene_psk_psa#30]
Hi.  Just a note.  Had no issues with 3.4, but unlike what the readme says, 2.83 is not quite supported.  The psa interface is broken in earlier versions, causing no 'import' button to appear:
```
Traceback (most recent call last):
  File "io_scene_psk_psa/psa/importer.py", line 558, in draw
    col = layout.column(heading='')
TypeError: UILayout.column(): was called with invalid keyword argument(s) (heading), expected (align)
```

**Comment by cmbasnett** [comment]
Thank you for the report; sorry for getting to this late. It seems that I added in a new API argument that doesn't exist in 2.83 but never noticed this issue because I only test with later versions.

Blender 2.83 is out of LTS support later this year anyways, so there's really not any reason for people to be using it over later versions.

I have bumped the minimum version from 2.83 to 2.90 in the latest release (4.2.1).
